suppress.xml

```diff
@@ -9,4 +9,54 @@
         <packageUrl regex="true">^pkg:maven/org\.springframework/spring\-web@.*$</packageUrl>
         <cve>CVE-2016-1000027</cve>
     </suppress>
+    <suppress until="2024-04-30Z">
+        <notes><![CDATA[
+   file name: sonar-scanner-api-2.16.2.588.jar (shaded: com.squareup.okhttp3:okhttp-urlconnection:3.14.2)
+   ]]></notes>
+        <packageUrl regex="true">^pkg:maven/com\.squareup\.okhttp3/okhttp\-urlconnection@.*$</packageUrl>
+        <cve>CVE-2023-0833</cve>
+    </suppress>
+    <suppress until="2024-04-30Z">
+        <notes><![CDATA[
+   file name: sonar-scanner-api-2.16.2.588.jar (shaded: com.squareup.okhttp3:okhttp:3.14.2)
+   ]]></notes>
+        <packageUrl regex="true">^pkg:maven/com\.squareup\.okhttp3/okhttp@.*$</packageUrl>
+        <vulnerabilityName>CVE-2021-0341</vulnerabilityName>
+    </suppress>
+    <suppress until="2024-04-30Z">
+        <notes><![CDATA[
+   file name: sonar-scanner-api-2.16.2.588.jar (shaded: com.squareup.okio:okio:1.17.2)
+   ]]></notes>
+        <packageUrl regex="true">^pkg:maven/com\.squareup\.okio/okio@.*$</packageUrl>
+        <cve>CVE-2023-3635</cve>
+    </suppress>
+    <suppress until="2024-04-30Z">
+        <notes><![CDATA[
+   file name: sonar-scanner-api-2.16.2.588.jar (shaded: com.squareup.okhttp3:okhttp:3.14.2)
+   ]]></notes>
+        <packageUrl regex="true">^pkg:maven/com\.squareup\.okhttp3/okhttp@.*$</packageUrl>
+        <cve>CVE-2023-0833</cve>
+    </suppress>
+    <suppress until="2024-04-30Z">
+        <notes><![CDATA[
+   file name: json-20231013.jar
+   ]]></notes>
+        <packageUrl regex="true">^pkg:maven/org\.json/json@.*$</packageUrl>
+        <cve>CVE-2022-45688</cve>
+    </suppress>
+    <suppress until="2024-04-30Z">
+        <notes><![CDATA[
+   file name: json-20231013.jar
+   ]]></notes>
+        <packageUrl regex="true">^pkg:maven/org\.json/json@.*$</packageUrl>
+        <cve>CVE-2023-5072</cve>
+    </suppress>
```

Comment:
perhaps I'm doing something wrong, but I don't see `json-20231013.jar` listed as vulnerability.  I threw some `xxxx`'s in the package regex and it didn't show when I ran the check.

Reply:
You're right, these json ones were added a bit earlier, but aren't required any more. I'll remove them.